include/OpenColorIO/OpenColorIO.h

```diff
@@ -1503,7 +1504,7 @@ class OCIOEXPORT Config
      * 
      * \return bool Archivable if true.
      */
-    bool isArchivable() const;
+    bool isArchivable(bool minimal) const;
```

Comment:
Personally I don't like the use of Flags as they can hide the actual meaning so I guess I defaulted to a single boolean, that said I'd prefer not using multiple bool's in APIs, so if we needed other parameters in the future then I'd suggest wanting to wrap the boolean in a stronger  type to avoid calling `function(true, false, true, ...)` but if others want Flags  That is simple enough

Reply:
I agree with you, although to me, even one boolean is not very explicit here, when you see `isArchivable(true)` somewhere randomly in the code. Flags are already used in other places of the API like for the Processor queries so that might be more consistent.